### ISSUE TITLE
Please include LICENSE in the npm package

### ISSUE THREAD

**Issue by bhj** [pradel/react-responsive-modal#518]
Thanks for all your work! Please include a LICENSE file in the npm package so we can properly credit you :)

```
WARNING in license-webpack-plugin: could not find any license file for react-responsive-modal.
```

**Comment by pradel** [comment]
The license is already published to npm in the package.json, see https://www.unpkg.com/browse/react-responsive-modal@6.4.1/package.json

**Comment by bhj** [comment]
No, `package.json` only specifies the license _type_. A `LICENSE` file should include the terms of the license and the information we need to comply with it. For example, part of the MIT license states:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Since there's no `LICENSE` file with a copyright notice and permission notice (the text of the license itself), it is not actually possible for us to comply with these terms, and the warning thrown by `license-webpack-plugin` is correct. Again, please include a proper `LICENSE` file in the package.
